build.gradle

```diff
@@ -408,7 +408,7 @@ jar {
 jib {
     extraDirectories.paths = ['build/gen-expanded']
     from {
-        image = "gcr.io/distroless/java11-debian11"
+        image = "us.gcr.io/broad-dsp-gcr-public/base/jre:11-debian"
```

Comment:
These really shouldn't be showing up in the diff - I think it just needs latest merged in?